crates/node-core/src/version.rs

```diff
@@ -11,6 +17,39 @@ pub const VERGEN_GIT_SHA: &str = env!("VERGEN_GIT_SHA");
 /// The build timestamp.
 pub const VERGEN_BUILD_TIMESTAMP: &str = env!("VERGEN_BUILD_TIMESTAMP");
 
+//The identification of the client
+/// - The two letter client code
+/// - The human readable name of the client
+/// - The version string of the current implementation
+/// - first four bytes of the latest commit hash of this build
+///
+/// # Example
+///
+///  ```text
+// {
+//   code: "RH",
+//   name: "Reth",
+//   version: "v0.2.0-beta.5",
+//   commit: "defa64b2"
+// }
+///  ```
+
+#[derive(Debug)]
+#[allow(missing_docs)]
+pub struct ClientVersionV1<'a> {
+    pub code: &'a str,
+    pub name: &'a str,
+    pub version: &'a str,
+    pub commit: &'a str,
+}
+#[allow(missing_docs)]
```

Comment:
remove this allow and add some docs:)

Reply:
Hey , can you please tell me where should i add the docs?

Reply:
sorry i was using // instead of /// , so i was getting this error.